docs/ref/macro.rst

```diff
@@ -3,86 +3,153 @@
 |                                                             |                                                                    |
-|    t`Refresh inbox`                                         |    /*i18n*/{                                                       |
+|    t`Refresh inbox`                                         |    i18n._(/*i18n*/{                                                |
 |                                                             |      id: "Refresh inbox"                                           |
-|                                                             |    }                                                               |
-|                                                             |                                                                    |
+|                                                             |    })                                                              |
 +-------------------------------------------------------------+--------------------------------------------------------------------+
 | .. code-block:: js                                          | .. code-block:: js                                                 |
 |                                                             |                                                                    |
-|    t`Attachment ${name} saved`                              |    /*i18n*/{                                                       |
+|    t`Attachment ${name} saved`                              |    i18n._(/*i18n*/{                                                |
 |                                                             |      id: "Attachment {name} saved",                                |
 |                                                             |      values: { name }                                              |
-|                                                             |    }                                                               |
-|                                                             |                                                                    |
+|                                                             |    })                                                              |
 +-------------------------------------------------------------+--------------------------------------------------------------------+
 | .. code-block:: js                                          | .. code-block:: js                                                 |
 |                                                             |                                                                    |
-|    plural({                                                 |    /*i18n*/{                                                       |
+|    plural({                                                 |    i18n._(/*i18n*/{                                                |
```

Comment:
on the new method is not like this?: 

```
js
plural(count, {
  one: "Message",
  other: "Messages"
})
```

On the codemod I have a method to rewrite this macro from { value: count ....} to (count, {...})

Reply:
You're right, I missed this code example 👍